low/low.h

```diff
@@ -811,6 +816,18 @@ bool check_hwtxchecksum_capability(uint16_t port_id) {
 	return (dev_info.tx_offload_capa & flags) == flags;
 }
 
+bool check_hwrxpackets_timestamp_capability(uint16_t port_id) {
```

Comment:
We have multiple functions that involves rte_eth_dev_info_get. It is better to have only one. However maybe it is in the future.